pyinsights/helper.py

```diff
@@ -129,5 +129,5 @@ def processing(msg: str, end: str = '') -> None:
     """
 
     processing_msg = f'{Accessory.Accent}{color()}{msg}{Accessory.End}{end}'
-    sys.stdout.write(processing_msg)
-    sys.stdout.flush()
+    sys.stderr.write(processing_msg)
+    sys.stderr.flush()
```

Comment:
@jdnrg 

This means that processing messages are treated as errors.
But this implicit not to write the message to stdout.
Is my guess right ?
If you don't want to write the message to stdout, I have a better idea this change.
My idea is to introduce a `-q, --quiet` option which specifies that no processing messages should be displayed.

Reply:
It is just a second channel, this allows for the json data to be redirected from stdout and the color output to remain on the stderr, which works great. 
I run now like this
```python3.8 -m pyinsights.cli -c ./examples/pyinsights_ssh_prod.yml --profile prod -r us-east-1 > results.json```

Reply:
Want to get only the resulting JSON data?

The processing message is not an error and should not be output to stderr.
In this case, It is better to introduce an option to suppress the processing of message output.

Reply:
https://en.wikipedia.org/wiki/Standard_streams#Standard_error_(stderr) 
`Standard error is another output stream typically used by programs to output error messages or diagnostics.`
this is diagnostics and stderr is perfect for it.

Reply:
Sorry.
I can't agree with your opinion, so close this pull request.

I want to implement the `--quiet` option to suppress the display of processing messages.